src/app/_components/landing.tsx

```diff
@@ -172,7 +172,7 @@ export default function Landing({ isVisible }: { isVisible: boolean }) {
         <div className="xl:pt-[200px] flex flex-col justify-center items-center w-[90%] xl:w-[60%] text-center">
           <h1 className="xl:leading-[1] text-3xl md:text-6xl font-[compasse-extrabold] leading-tight">THE JOINT EMERGENCY ACTION PLAN (JEAP)</h1>
           <h1 className="xl:leading-[1] text-3xl md:text-6xl font-[compasse-extrabold] leading-tight">UNLOCKING AFRICA'S RESILIENCE</h1>
-          <p className="text-lg md:text-2xl pt-5 w-[80%] break-normal">
+          <p className="text-lg md:text-2xl w-[100%] sm:leading-none sm:px-4 md:w-[80%] xl:w-[80%] text-balance break-normal">
```

Comment:
There is an existing padding top
Why have we removed that?

it should still be there for above md devices right?

Reply:
Updated